packages/taquito-rpc/src/types.ts

```diff
@@ -1499,6 +1500,23 @@ export interface ConstantsResponseCommon {
 
 export type Ratio = { numerator: number; denominator: number };
 
+export interface TicketUpdates {
+  ticket_token: {
+    ticketer: string;
+    content_type: MichelsonV1Expression;
+    content: MichelsonV1Expression;
+  };
+  updates: {
+    account: string;
+    amount: BigNumber;
```

Comment:
If amount is set as a BigNumber, it needs to be cast to that type in the corresponding rpc method. It is probably simpler to set it as a string.